From dba0257d49269a7aa49319ae43ea69b539000123 Mon Sep 17 00:00:00 2001
From: asadgits <iamasadiqbal@gmail.com>
Date: Mon, 26 Jul 2021 12:49:53 +0500
Subject: [PATCH 10/12] main method changes

---
 ThirdTask/src/MainClass.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ThirdTask/src/MainClass.java b/ThirdTask/src/MainClass.java
index b91b365..5bc0ce4 100644
--- a/ThirdTask/src/MainClass.java
+++ b/ThirdTask/src/MainClass.java
@@ -57,7 +57,7 @@ public class MainClass {
 
 
 
-
+    
 
 
 
-- 
2.32.0.windows.2

